libpromises/eval_context.c

```diff
@@ -765,7 +765,7 @@ void EvalContextHeapPersistentLoadAll(EvalContext *ctx)
         {
             Log(LOG_LEVEL_VERBOSE, "Persistent class '%s' for %jd more minutes",
                 key, (intmax_t) ((info.expires - now) / 60));
-            Log(LOG_LEVEL_VERBOSE, "Adding persistent class '%s' to heap", key);
+            Log(LOG_LEVEL_DEBUG, "Adding persistent class '%s' to heap", key);
```

Comment:
If classes are not stored as a heap  https://en.wikipedia.org/wiki/Heap_(data_structure) we should probably use a different word here. I believe that when originally authored Mark Burgess used definition 1 for heap: "an untidy collection of things piled up haphazardly." That's what he did for class as well: "a set or category of things having some property or attribute in common and differentiated from others by kind, type, or quality."

Reply:
Yes, please just remove any mention of data structure type. "Adding persistent class '%s'" might be nice.